src/controller/user/routes.ts

```diff
@@ -1,9 +1,10 @@
-import { FastifyInstance } from 'fastify'
-import { getUserById } from './getUserById'
-import { getUserByEmail } from './getUserByEmail'
+import { FastifyInstance } from "fastify";
+import { getUserById } from "./getUserById";
+import { getUserByEmail } from "./getUserByEmail";
+import { registerUser } from "./registerUser";
 
 export async function userRoutes(app: FastifyInstance) {
-
-  app.get('/user/:id', getUserById)
-  app.get('/user', getUserByEmail)
+  app.post("/users", registerUser);
```

Comment:
Why the route is called users? We are just creating one user and also, the standard that we are using is singular "user"